package/yast2-partitioner.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Tue Jun 27 08:20:22 UTC 2017 - mvidner@suse.com
+
+- A proper partition size dialog
```

Comment:
well, it sounds a bit strange to mark dialog as proper. What about `Modify partition size dialog to have all features from old partitioner`?